content/docs/get-started/aws/modify-program.md

```diff
@@ -93,38 +93,38 @@ pulumi.export('bucket_name',  bucket.id)
 package main
 
 import (
-    "github.com/pulumi/pulumi-aws/sdk/go/aws/kms"
-    "github.com/pulumi/pulumi-aws/sdk/go/aws/s3"
-    "github.com/pulumi/pulumi/sdk/go/pulumi"
+	"github.com/pulumi/pulumi-aws/sdk/go/aws/kms"
```

Comment:
FWIW, I think @cnunciato intentionally asked to remove the tabs, since we generally don't use them in our code snippets. But I get that in Go this is idiomatic. Just calling it out since he had commented on this previously.

Reply:
Spaces will behave much more reliably than tabs, yeah. 🙏

Reply:
Tabs really are the right choice here if we want these examples to be copy-pastable and idiomatically formatted (i.e. `gofmt` clean). Code samples in Go docs always use tabs (see e.g. the examples in https://golang.org/pkg/os/).

Reply:
I do see that point, re: copy-paste, yeah. 🤔 Let's try to make it work with tabs, then -- I do suspect we're going to have formatting issues, but we can address them when the time comes.